src/reactable.jsx

```diff
@@ -344,6 +344,7 @@
             var Ths = [];
             for (var index = 0; index < this.props.columns.length; index++) {
                 var column = this.props.columns[index];
+                var thClass = column.key
```

Comment:
Awesome! I forgot the case when user can leave the label with whitespace. So this should be `'reactable-th-' + column.key.replace(/ /g, '-').toLowerCase()`?